spec/request/vet360/telephone_request_spec.rb

```diff
@@ -0,0 +1,159 @@
+        VCR.use_cassette('vet360/contact_information/post_telephone_success') do
+          post(
+            '/v0/profile/telephones',
+            telephone.to_json,
+            auth_header.update(
+              'Content-Type' => 'application/json', 'Accept' => 'application/json'
+            )
+          )
+
+          expect(response).to have_http_status(:ok)
+          expect(response).to match_response_schema('vet360/transaction_response')
+        end
+      end
+
+      it 'creates a new AsyncTransaction::Vet360::TelephoneTransaction db record' do
+        VCR.use_cassette('vet360/contact_information/post_telephone_success') do
+          expect do
+            post(
+              '/v0/profile/telephones',
+              telephone.to_json,
+              auth_header.update(
+                'Content-Type' => 'application/json', 'Accept' => 'application/json'
+              )
+            )
+          end.to change(AsyncTransaction::Vet360::TelephoneTransaction, :count).from(0).to(1)
+        end
+      end
+    end
+
+    context 'with a 400 response' do
+      it 'should match the errors schema', :aggregate_failures do
+        telephone.id = 42
+
+        VCR.use_cassette('vet360/contact_information/post_telephone_w_id_error') do
+          post(
+            '/v0/profile/telephones',
+            telephone.to_json,
+            auth_header.update(
+              'Content-Type' => 'application/json', 'Accept' => 'application/json'
+            )
+          )
+
+          expect(response).to have_http_status(:bad_request)
+          expect(response).to match_response_schema('errors')
+        end
+      end
+    end
+
+    context 'with a 403 response' do
+      it 'should return a forbidden response' do
+        VCR.use_cassette('vet360/contact_information/post_telephone_status_403') do
+          post(
+            '/v0/profile/telephones',
+            telephone.to_json,
+            auth_header.update(
+              'Content-Type' => 'application/json', 'Accept' => 'application/json'
+            )
+          )
+
+          expect(response).to have_http_status(:forbidden)
+        end
+      end
+    end
+
+    context 'with a validtion issue' do
+      it 'should match the errors schema', :aggregate_failures do
+        telephone.phone_number = ''
+
+        post(
+          '/v0/profile/telephones',
+          telephone.to_json,
+          auth_header.update(
+            'Content-Type' => 'application/json', 'Accept' => 'application/json'
+          )
+        )
+
+        expect(response).to have_http_status(:unprocessable_entity)
+        expect(response).to match_response_schema('errors')
+        expect(errors_for(response)).to include "phone-number - can't be blank"
+      end
+    end
+  end
+
+  describe 'PUT /v0/profile/telephones' do
+    let(:telephone) { build(:telephone, vet360_id: user.vet360_id) }
+
+    before { telephone.id = 42 }
+
+    context 'with a 200 response' do
```

Comment:
This isn't a big deal, but possibly worth mentioning... I realized after I wrote this `context` name that I was probably doing it wrong. The "Self-documentingness" works better if we're dutiful about what's a context and what's an assertion, etc. Here, I think `with a 200 response` is definitely not a context. It's more like the result. The context is actually probably something like "when a valid request is made".